src/Tizen.NUI/src/public/BaseComponents/ViewAccessibility.cs

```diff
@@ -285,15 +183,44 @@ public void ClearAccessibilityRelations()
         /// <summary>
         /// Gets accessibility collection connected with the current object.
         /// </summary>
-        /// <returns>AddressCollection</returns>
+        /// <returns>A dictionary mapping a relation type to a set of objects in that relation</returns>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public AddressCollection GetAccessibilityRelations()
+        public Dictionary<AccessibilityRelationType, List<View>> GetAccessibilityRelations()
         {
-            var result = new AddressCollection(Interop.ControlDevel.DaliToolkitDevelControlNewGetAccessibilityRelations(SwigCPtr));
+            var list = new List<KeyValuePair<int, IntPtr>>();
+            var listHandle = GCHandle.Alloc(list);
+            var callback = new Interop.ControlDevel.GetAccessibilityRelationsCallback(GetAccessibilityRelationsCallback);
+
+            Interop.ControlDevel.DaliToolkitDevelControlGetAccessibilityRelations(SwigCPtr, callback, GCHandle.ToIntPtr(listHandle));
+            listHandle.Free();
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+
+            var result = new Dictionary<AccessibilityRelationType, List<View>>();
+
+            foreach (var pair in list)
+            {
+                var key = (AccessibilityRelationType)pair.Key;
+                var value = this.GetInstanceSafely<View>(pair.Value);
+
+                if (!result.ContainsKey(key))
+                {
+                    result[key] = new List<View>();
+                }
+
+                result[key].Add(value);
+            }
+
             return result;
         }
 
+        private static void GetAccessibilityRelationsCallback(int relationType, IntPtr relationTarget, IntPtr userData)
+        {
+            var handle = GCHandle.FromIntPtr(userData);
+            var list = (List<KeyValuePair<int, IntPtr>>)handle.Target;
+
+            list.Add(new KeyValuePair<int, IntPtr>(relationType, relationTarget));
+        }
+
```

Comment:
How did you get this idea? Very GR8!! Thumbs UP!!
We have a ```TizenFX/test/Tizen.NUI.Samples/Tizen.NUI.Samples/Samples/AtspiSample.cs```
It would be better we have an example code for this. It could be separated a PR.

Reply:
> How did you get this idea?

The idea of using a single interop with a callback function called for each item in the collection is based on this code example for `GCHandle` and `EnumWindows`: https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.gchandle?view=net-6.0. The advantage is that there is only one interop needed (the previous version with `AddressCollection` required many).

> It would be better we have an example code for this. It could be separated a PR.

OK, I'll add that to my TODO list. BTW, we are also developing more Aurum tests for relations.